lib/gltf.js

```diff
@@ -154,185 +43,301 @@ function createGltf(data, inputPath, modelName) {
     };
 
     gltf.asset = {
-        "generator": "OBJ2GLTF",
-        "premultipliedAlpha": true,
-        "profile": {
-            "api": "WebGL",
-            "version": "1.0"
+        generator : 'obj2gltf',
+        profile : {
+            api : 'WebGL',
+            version : '1.0'
         },
-        "version": 1
+        version: '1.0'
     };
 
     gltf.scenes[sceneId] = {
-        nodes : [nodeId]
+        nodes : []
     };
 
-    gltf.nodes[nodeId] = {
-        children : [],
-        matrix : [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1],
-        meshes : [meshId],
-        name : modelName
-    };
+    function getImageId(imagePath) {
```

Comment:
Is there a reason not to use the entire `imagePath` as the id?

We should avoid "this is a bug but unlikely to happen" scenarios, it's just a longer way to say "this is a bug".  Assuming we can't use the entire image path, an easy way to address this is to have a cache that maps paths to ids that lives as long as the createGltf call.